src/dotty/tools/dotc/config/Config.scala

```diff
@@ -15,11 +15,12 @@ object Config {
    */
   final val checkConstraintsNonCyclic = false
 
-  /** Like `checkConstraintsNonCyclic`, but all constrained parameters
-   *  are tested for direct or indirect dependencies, each time a
-   *  constraint is added in TypeComparer.
+  /** Make sure none of the bounds in an OrderingConstraint contains
+   *  another constrained parameter at its toplevel (i.e. as an operand
```

Comment:
You say "another constrained parameter" but the check does  `b eq param`, so shouldn't you say something like "Make sure none of the bounds of a parameter in an OrderingConstraint contains this parameter at its toplevel" ?